includes/admin/class-wcmypa-admin.php

```diff
@@ -156,7 +156,14 @@ public function addDeliveryDayFilterToOrdergrid(): void
      */
     public function deliveryDayFilter(): void
     {
-        if (is_admin() && ! empty($_GET['post_type']) == 'shop_order') {
+        $postNlShowDate   = WCMYPA()->setting_collection->where('carrier', CarrierPostNL::NAME)
```

Comment:
Losse postNLShowDate en instaboxShowDate vind ik eng, is het mogelijk een method `anyActiveCarrierHasShowDeliveryDate` (ofzo) te maken die door de active carriers loopt en dan true returned as er 1 show_delivery_date heeft? En anders false?